cluster-autoscaler/processors/nodegroupset/clusterapi_nodegroups.go

```diff
@@ -25,11 +25,13 @@ import (
 // even if they have different infrastructure provider-specific labels.
 func CreateClusterAPINodeInfoComparator(extraIgnoredLabels []string) NodeInfoComparator {
 	capiIgnoredLabels := map[string]bool{
-		"topology.ebs.csi.aws.com/zone":                 true, // this is a label used by the AWS EBS CSI driver as a target for Persistent Volume Node Affinity
-		"topology.diskplugin.csi.alibabacloud.com/zone": true, // this is a label used by the Alibaba Cloud CSI driver as a target for Persistent Volume Node Affinity
-		"ibm-cloud.kubernetes.io/worker-id":             true, // this is a label used by the IBM Cloud Cloud Controler Manager
+		"ibm-cloud.kubernetes.io/vpc-instance-id":       true, // used on IBM Cloud when a VPC is in use
```

Comment:
can we may be expose them from each providers and reference them here so we don't need to manually keep the list?

Reply:
i would be ok with modifying the other nodegroupset processors to expose the variables as constants there, would that work?

Reply:
created #5158

Reply:
changed to use consts from other providers where applicable